packages/fxa-payments-server/server/config/index.js

```diff
@@ -177,6 +177,15 @@ const conf = convict({
       },
     },
   },
+  oauth_client_id_map: {
```

Comment:
The snake_case here is consistent with the content-server, from where this config is lifted; it is inconsistent with the camelCase used for the other config names in this file.